modules/server/src/network/types.ts

```diff
@@ -0,0 +1,13 @@
+import { GraphQLSchema } from 'graphql';
+
+// environment config
+export type NetworkAggregationConfigInput = {
+	graphqlUrl: string;
+	documentType: string;
+	displayName: string;
+};
+
+// additional properties after initialising network
+export type NetworkAggregationConfig = NetworkAggregationConfigInput & {
+	schema?: GraphQLSchema;
+};
```

Comment:
This is a helpful comment but it wasn't easily understandable. I had to read the type information to understand what you mean. Maybe something like:


```suggestion
/**
 * Complete aggregation config that defines the network search setup. This includes the original
 * config information plus computed fields that are generated in the network search initialization process.
 */
export type NetworkAggregationConfig = NetworkAggregationConfigInput & {
	schema?: GraphQLSchema;
};
```

Reply:
That's a nice improvement, for sure!
Naive question: am I the only one who gets bugged by the line length inconsistency?
personally, I'd chop it like this
```
* Complete aggregation config that defines the network search setup.
* This includes the original config information plus computed fields
* that are generated in the network search initialization process.
```